tests/test_subdirectory.py

```diff
@@ -155,165 +173,156 @@ def test_update_subdirectory_from_root_path(tmp_path_factory):
 @pytest.mark.parametrize(
     "conflict, readme, expect_reject",
     [
-        pytest.param(
+        (
             "rej",
             "upstream version 2\n",
             True,
         ),
-        pytest.param(
+        (
             "inline",
-            "<<<<<<< before updating\ndownstream version 1\n"
-            "=======\nupstream version 2\n>>>>>>> after updating\n",
+            dedent(
+                """\
+                <<<<<<< before updating
+                downstream version 1
+                =======
+                upstream version 2
+                >>>>>>> after updating
+                """
```

Comment:
Do you agree with using a multiline string for the inline merge conflict? I think it's much more readable. The line breaks are implicit, yes, but it's much more obvious what the merge conflict syntax is.

Reply:
OK, let's keep this one.